tests/t400-qfunction-f.okl

```diff
@@ -18,18 +18,17 @@
 typedef int CeedInt;
 typedef double CeedScalar;
 // OCCA parser doesn't like __global here
-typedef /*__global*/ double gCeedScalar;
+//typedef __global double gCeedScalar;
```

Comment:
Commented-out dead code smells bad.

Reply:
Borked change and revert. I'll track down all versions of this.